packages/client-api-schema/src/validation.ts

```diff
@@ -18,7 +18,9 @@ const apiSchema = require('./generated-schema.json'); // because https://github.
 ajv.addSchema(apiSchema, 'api.json');
 
 export const validateRequest = ajv.compile({$ref: 'api.json#/definitions/StateChannelsRequest'});
-export const validateResponse = ajv.compile({$ref: 'api.json#/definitions/StateChannelsResponse'});
+export const validateResponse = ajv.compile({
+  $ref: 'api.json#/definitions/StateChannelsResponse'
+});
```

Comment:
I'm confused by this change. Has the line length changed?

Reply:
The `printWidth` option has not changed, and remains at 100. This is [not a hard and fast line length limit](https://prettier.io/docs/en/options.html#print-width) in prettier, though. 

There's also this https://prettier.io/blog/2020/03/21/2.0.0.html#improved-method-chain-breaking-heuristic-6685httpsgithubcomprettierprettierpull6685-by-mmkalhttpsgithubcommmkal

which might be causing it. Overall I am not sure.

Reply:
This line is 100 chars long. So it should in fact be broken.  Not sure why it wasn't before.